style-guide/static/handover-process.js

```diff
@@ -0,0 +1,131 @@
+  "phase5.subheader1": "Communicate if updates need to be made",
+  "phase5.description1": "After launch you’ll learn new things based on usage and feedback from users. If there are updates, document changes in Jira so that engineers can prioritize with other tasks. Where relevant, provide customer feedback with tickets to help provide context.",
+
+  "quote1": "The storymapping exercise was so beneficial for expectation setting with the product development team. It becomes a delivery tool, supports release planning and creates a shared discussion about workflows without needing to create designs.",
+  "quote1.user": "Jake Clements, Engineering Manager",
+
+  "quote2": " Open communication is critical to the product development process. Finding the preferred way of keeping track of design or copy changes (whether that is Jira, Figma or Slack) upfront avoids confusion down the road.",
+  "quote2.user": "Briana Lee, Product Designer",
+
+  "resource1.title": "Product brief template",
+  "resource1.description": "There is no need to start from scratch. Use this template to get a project started.",
+  "resource1.linkTitle": "View the product brief template",
+  "resource1.link": "https://google.com",
+
+  "resource2.title": "Learn how to Story Map",
+  "resource2.description": "Story mapping is a pattern. It’s how you make sense of a whole product or feature. You do so by breaking down large stories into smaller ones.",
+  "resource2.linkTitle": "View the guide",
+  "resource2.link": "https://google.com",
+
+  "resource3.title": "Critiquing designs",
+  "resource3.description": "Critique builds shared perspectives and values, leverages the strength of the entire team, and drives iteration and improvement.",
+  "resource3.linkTitle": "Learn how to critique",
+  "resource3.link": "https://google.com",
+
+  "resource4.title": "Product content process",
+  "resource4.description": "Manage the way you collaborate and share final content with stakeholders with this guide.",
+  "resource4.linkTitle": "View the guide and template",
+  "resource4.link": "https://google.com",
+
+  "resource5.title": "Buffet library",
+  "resource5.description": "View all the components that exist today in Buffet Storybook. These are components used throughout the restaurant product.",
+  "resource5.linkTitle": "View Buffet components",
+  "resource5.link": "https://google.com",
+
+  "resource6.title": "Decision Tree",
+  "resource6.description": "Need help prioritizing your features? Use the decision tree template to move your project along.",
+  "resource6.linkTitle": "View the decision tree",
+  "resource6.link": "https://google.com",
+
+  "resource7.title": "Quality assurance template",
+  "resource7.description": "Want to maintain consistency and quality as you ship new products and features? Use this template to keep track of all issues.",
+  "resource7.linkTitle": "View the QA template",
+  "resource7.link": "https://google.com"
```

Comment:
Code is looking good to go, do we need to get links for these resources before we release though?